game-core/src/main/java/games/strategy/triplea/delegate/power/calculator/AirBattleDefenseCombatValue.java

```diff
@@ -0,0 +1,93 @@
+package games.strategy.triplea.delegate.power.calculator;
+
+import games.strategy.engine.data.GameData;
+import games.strategy.engine.data.TerritoryEffect;
+import games.strategy.engine.data.Unit;
+import java.util.Collection;
+import java.util.List;
+import java.util.Map;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NonNull;
+import lombok.Value;
+import org.triplea.java.collections.IntegerMap;
+
+/** Calculates offense strength and roll for Air Battle dice */
+@Builder
+@Value
+@Getter(AccessLevel.NONE)
+class AirBattleDefenseCombatValue implements CombatValue {
+
+  @Getter(onMethod = @__({@Override}))
+  @NonNull
+  GameData gameData;
+
+  @Getter(onMethod = @__({@Override}))
+  @NonNull
+  @Builder.Default
+  Collection<Unit> friendUnits = List.of();
+
+  @Getter(onMethod = @__({@Override}))
+  @NonNull
+  @Builder.Default
+  Collection<Unit> enemyUnits = List.of();
+
+  @Getter(onMethod = @__({@Override}))
```

Comment:
minor, it can be a nicer syntax to use the 'underscore' variant, eg: `@Getter(onMethod_ = @Override)`